models/abstract.js

```diff
@@ -17,16 +17,28 @@ exports.create = function(model, values, res, callback) {
 		exports.process(values, values => {
 			db.get().query('INSERT INTO '+model+' SET ?', values, 
 				(err, results) => {
-					if (res) httpResponse(err, 400, results, 201, res);
-					else noHttpResponse(err, results, callback);
+					if (res) { callback(null, values); }
```

Comment:
This is fine but I think we might want to agree on some standards just to keep the code consistent. Between line's 20:39, there are 3 different ways of writing an if statement, It's kinda funny.

if (something) {yes();}
else {no();}

if (something) yes();
else no();

if (something) {
  yes();
}
else {
  no();
}

Reply:
That is a good point. I'll have to bring it up during online meetings. The reason why I would still keep curly braces on the one liners is for the ease of putting console.log statements (I don't have a better debugging strategy).